examples/simple.js

```diff
@@ -1,7 +1,7 @@
 /* global Realm */
 /* eslint-disable no-self-compare, no-console */
 
-const r = Realm.makeRootRealm();
+const r = Realm.default.makeRootRealm();
```

Comment:
Why is this required to introduce configurable transforms?

Reply:
As you probably have noticed there weren't any breaking changes introduced in the shim's code. The only thing I've added was a named export. If there were any such changes the entire suite of tests would have had to be updated, but they were not. 

Unfortunately UMD packages do not like having a default export and a named export simultaneously and this is the effect of that. Regular ES6 modules are not affected. The example can be updated to use ES6 modules rather than regular scripts (`<script type="module" ../>` vs `<script type="text/javascript../>`) and this "issue" would not be seen anymore. I didn't do that because I felt it's important to see how to use both the UMD and the ES6 module.